app/models/certificate.rb

```diff
@@ -0,0 +1,7 @@
+# generic polymorphic model for all kinds of certificates
+# rubocop:disable Rails/HasManyOrHasOneDependent
+class Certificate < ActiveRecord::Base
+  has_many :docker_registries, as: :certifiable
```

Comment:
Is this right? Isn't it the other way around? A certificate belongs to a certifiable entity and this certifiable entity has one certificate? A certificate should not know about docker registry since it's a polymorphic model. Does it make sense ?

I took a look at http://guides.rubyonrails.org/v4.2/association_basics.html#polymorphic-associations and compared with their scenario and it felt wrong for me the way we are doing it.